iothub_service_client/tests/iothub_msging_ll_ut/iothub_messaging_ll_ut.c

```diff
@@ -1453,76 +1453,6 @@ BEGIN_TEST_SUITE(iothub_messaging_ll_ut)
         IoTHubMessaging_LL_Destroy(iothub_messaging_handle);
     }
 
-#if 0
```

Comment:
I'm 100% up for removing everything that has been disabled like that without a justification, and if we verify the tests are really not needed. If they are we maybe should file a task to re-add them appropriately.

Reply:
100% agree - if these tests were needed/intereting I would've re-enabled or created an IOU to bring them back.

This case we're OK though.

`IoTHubMessaging_LL_SendModule` is an API we'd considered back in the day but removed prior to GA and it's not something I see coming back for many reasons (both for IoT Hub philosophy & because we don't add new things to service SDK for C).

Thanks for thorough review.